### ISSUE TITLE
No assistant found with id 'asst_4Gvk7WJuvcSqPuzUciiGSw8j'

### ISSUE THREAD

**Issue by vasco** [felipetodev/wanderlust#1]
Hey!

After you setup the project and try to run it locally, you get the message (server logs):

> Error: 404 No assistant found with id 'asst_4Gvk7WJuvcSqPuzUciiGSw8j'.
>     at async runAssistant (./src/app/actions/run-assistant.ts:27:17)

Should we create our own Assistant in https://platform.openai.com/ ?

**Comment by felipetodev** [comment]
Hi!

Replace the assistant here: https://github.com/felipetodev/wanderlust/blob/main/src/app/actions/run-assistant.ts#L7

and then add your functions like:
![image](https://github.com/felipetodev/wanderlust/assets/69825873/069c4ade-7db6-433d-943d-c77c97e9fea2)

BTW. The assistant ID: `asst_4Gvk7WJuvcSqPuzUciiGSw8j` is actually running with that functions but I think you can't use it with any OpenAI key other than mine. If you need the functions in detail, let me know, and I'll send them to you @vasco.

**Comment by vasco** [comment]
Correct, I can't access that with my OpenAI key. Can you share the assistant instructions and function descriptions?
